### ISSUE TITLE
Unhandled error when decorating iTunes result without an image URL

### ISSUE THREAD

**Issue by coderigo** [bobby-brennan/rss-parser#35]
`parser.parseURL()` throws the following error for podcast rss feeds that don't have an image URL:
```
Cannot read property 'href' of undefined
```

This happens occasionally and I've got about 10 podcasts for which I can reproduce. I've forked and fixed by just not adding an image property to `itunes` during decoration. I'll send a PR shortly to fix if it looks 👌 .

**Comment by bobby-brennan** [comment]
Fixed by #36

**Comment by bbrinx** [comment]
I am having the same problem. 

> TypeError: Cannot read property 'href' of undefined

The error occurs when calling `parser.parseURL('http://www.transfermarkt.de/rss/news')`
______
```
[0] TypeError: Cannot read property 'href' of undefined
[0]     at Parser.buildRSS (/path/node_modules/rss-parser/lib/parser.js:181:72)
[0]     at Parser.buildRSS2 (/path/node_modules/rss-parser/lib/parser.js:169:21)
[0]     at xmlParser.parseString (/path/node_modules/rss-parser/lib/parser.js:41:23)
[0]     at Parser.<anonymous> (/path/node_modules/xml2js/lib/parser.js:303:18)
[0]     at Parser.emit (events.js:182:13)
[0]     at SAXParser.onclosetag (/path/node_modules/xml2js/lib/parser.js:261:26)
[0]     at emit (/path/node_modules/sax/lib/sax.js:624:35)
[0]     at emitNode (/path/node_modules/sax/lib/sax.js:629:5)
[0]     at closeTag (/path/node_modules/sax/lib/sax.js:889:7)
[0]     at SAXParser.write (/path/node_modules/sax/lib/sax.js:1436:13)
```

**Comment by bobby-brennan** [comment]
Thanks for the report, fixed in 3.6.2

**Comment by bbrinx** [comment]
Thanks for the quick fix!
